instagram-app/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './App.css';
+import dummyData from './dummy-data'
+import SearchBar from './components/SearchBar/SearchBar'
+import PostContainer from './components/PostContainer/PostContainer'
+import CommentSection from './components/CommentSection/CommentSection';
+
+class App extends React.Component {
+  state = {
+    data: dummyData,
+  };
```

Comment:
Love to see you using the shorthand syntax for your state! It's so much cleaner and concise!